gui/qt/main_window.py

```diff
@@ -2206,7 +2206,7 @@ def do_pay_invoice(self, key):
 
     def create_console_tab(self):
         from .console import Console
-        self.console = console = Console()
+        self.console = console = Console(parent=self)
```

Comment:
Hmm. The parent eventually ends up becoming the QTabWidget (`main_window.py` `self.tabs`) -- is this necessary?

Reason I ask is.. it breaks with the "dumb way" they did it before (ie no initial parent) so.. that can cause bugs down the road if the dumb way ends up being something we rely on...

Reply:
This is the same how the history list gets its wallet instance.

Reply:
What would be a better way to get the wallet instance?

Reply:
Oh you needed the wallet instance.  IDK, then.  The thing that makes me nervous is parent changes later so that inconsistency irks me.  Why not just pass the wallet instance directly?